src/components/Recipe.js

```diff
@@ -19,27 +19,25 @@ export default function Recipe({
       <p dangerouslySetInnerHTML={{ __html: text }}></p>
       <h3>Missing Ingredients</h3>
       <ul aria-label="missing ingredients">
```

Comment:
you could use the aria-labelledby='list-title' here and give the h3 id='list-title' 🙂